src/dashboard/Data/Browser/Browser.react.js

```diff
@@ -79,8 +79,12 @@ class Browser extends DashboardView {
 
       isUnique: false,
       uniqueField: null,
+<<<<<<< HEAD
```

Comment:
Can you please resolve the merge conflicts? It seems that the file is merged, but the code lines are not properly merged.